src/utils.ts

```diff
@@ -126,14 +117,17 @@ export const delay = (ms: number): Promise<void> => new Promise(res => setTimeou
 export enum PondVersions {
   Version1 = 1,
   Version2 = 2,
+  Version3 = 3,
 }
-export const defaultPondVersions = PondVersions.Version2
+export const defaultPondVersions = PondVersions.Version3
```

Comment:
Should be singular: `defaultPondVersion`

Reply:
done